quickstarts/maven/hello-world/README.md

```diff
@@ -11,85 +11,57 @@ line and exits. We would be using Eclipse JKube for building a docker image and
+```
+3. Run the following command to run helloworld sample: 
 ```
 ~/work/repos/jkube/quickstarts/maven/hello-world : $ mvn clean install k8s:build k8s:resource k8s:apply
 [INFO] Scanning for projects...
 [INFO] 
-[INFO] ---------< org.eclipse.jkube.samples:jkube-sample-helloworld >----------
-[INFO] Building jkube-sample-helloworld 0.1.1-SNAPSHOT
+[INFO] -----------< org.eclipse.jkube.quickstarts.maven:helloworld >-----------
+[INFO] Building Eclipse JKube :: Quickstarts :: Maven :: Hello World 1.16.1
+[INFO]   from pom.xml
 [INFO] --------------------------------[ jar ]---------------------------------
 [INFO] 
-[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ jkube-sample-helloworld ---
-[INFO] 
-[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ jkube-sample-helloworld ---
-[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
-[INFO] skip non existing resourceDirectory /home/rohaan/work/repos/jkube/quickstarts/maven/hello-world/src/main/resources
-[INFO] 
-[INFO] --- maven-compiler-plugin:3.6.1:compile (default-compile) @ jkube-sample-helloworld ---
-[INFO] Changes detected - recompiling the module!
-[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
-[INFO] Compiling 1 source file to /home/rohaan/work/repos/jkube/quickstarts/maven/hello-world/target/classes
-[INFO] 
-[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ jkube-sample-helloworld ---
-[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
-[INFO] skip non existing resourceDirectory /home/rohaan/work/repos/jkube/quickstarts/maven/hello-world/src/test/resources
-[INFO] 
-[INFO] --- maven-compiler-plugin:3.6.1:testCompile (default-testCompile) @ jkube-sample-helloworld ---
-[INFO] Changes detected - recompiling the module!
-[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!
-[INFO] Compiling 1 source file to /home/rohaan/work/repos/jkube/quickstarts/maven/hello-world/target/test-classes
-[INFO] 
-[INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ jkube-sample-helloworld ---
-[INFO] Surefire report directory: /home/rohaan/work/repos/jkube/quickstarts/maven/hello-world/target/surefire-reports
-
--------------------------------------------------------
- T E S T S
--------------------------------------------------------
-Running org.eclipse.jkube.sample.helloworld.AppTest
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.044 sec
-
-Results :
-
-Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
-
-[INFO] 
-[INFO] --- maven-jar-plugin:3.0.2:jar (default-jar) @ jkube-sample-helloworld ---
-[INFO] Building jar: /home/rohaan/work/repos/jkube/quickstarts/maven/hello-world/target/jkube-sample-helloworld-0.1.1-SNAPSHOT.jar
+[INFO] --- k8s:1.16.1:build (default-cli) @ helloworld ---
```

Comment:
I think maybe we should remove these plugin logs as they become outdated with newer plugin releases. Just specifying the command should suffice